docs/sources/about.md

```diff
@@ -76,14 +84,12 @@ You should run the Static mode Kubernetes operator when:
 
 ### Flow mode
 
-[Flow mode][] is a stable variant of Grafana Agent first introduced on
+[Flow mode][] is a stable variant of Grafana Agent, introduced on
 September 29, 2022.
 
-Flow mode was introduced as a re-imagining of Grafana Agent with a focus on
-vendor neutrality, ease-of-use, improved debuggability, and ability to adapt to
-the needs of power users by adopting a configuration-as-code model.
-
-Flow mode is considered to be the future of the Grafana Agent project.
+Grafana Agent Flow mode focuses on vendor neutrality, ease-of-use,
+improved debuggability, and ability to adapt to the needs of power
+users by adopting a configuration-as-code model.
```

Comment:
nit:

Vale doesn't recognize "debuggability" as a word. Perhaps "easier debugging"? I guess this is a comparison to static mode. Should that be more clear?

nit:

[Semantic line breaks](https://sembr.org/) are better for future line based diffing in the GitHub UI and result in the same HTML output.

Reply:
Yeah I didn't clean up much in this pass (like debuggability). There's a lot of rewrite that's needed.

The semantic linebreaks are... difficult to manage because everyone has different linebreaks based on what tool they are using. Every edit ends up with wildly different linebreaks. Not sure how to manage/enforce this long term.

Reply:
Yeah, the argument of semantic line breaks is that they provide the most meaningful diff when using a line based diff. GitHub primarily works with line based diffing and that means that semantic line breaks provide the most easy to read diff in the primary place where we view patches.

Converesely, arbitrary line lengths are maybe sympathetic to those reading in a terminal but I would expect that any modern text editor or other presentation layer should be able to perform soft wrapping of long lines.